### PR TITLE
add keys

### PR DESCRIPTION
**Pull request by guillermooo**

### CODE REVIEW

**Comment by guillermooo** [comment]
@jpoon Not sure what's up with Travis. Looks like a connectivity issue.

**Comment by jpoon** [comment]
I added a license in the package.json. Weird its complaining on your branch and not mine.

File: src/mode/mode_handler.ts [hunk 1]
```diff
@@ -29,15 +29,18 @@ export default class ModeHandler {
...
     }
 
-    public SetCurrentModeByName(modeName : ModeName) {
+    SetCurrentModeByName(modeName : ModeName) {
         this.modes.forEach(mode => {
             mode.IsActive = (mode.Name === modeName);
         });
 
-        this.setupStatusBarItem(ModeName[modeName].toUpperCase());
+        // Vim never shows -- NORMAL -- in the status bar.
```
Comment (jpoon) [inline src/mode/mode_handler.ts]: I put it there intentionally just to make it more obvious, but we can remove it.

**Comment by jpoon** [comment]
Thanks! I'll merge it and we can fix any issues that arise in `insert-mode` branch directly.

**Pull request merged**